Ark.Tools.AspNetCore.HealthChecks/Ark.Tools.AspNetCore.HealthChecks.csproj

```diff
@@ -0,0 +1,42 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<!-- the other props are defined in Directory.Build.props -->
+
+	<!--===================Configuration for Nuget Package=========================-->
+	<PropertyGroup>
+
+		<TargetFrameworks>netcoreapp3.1</TargetFrameworks>
+
+		<Description>Extensions of AspNetCore HealthChecks</Description>
+		<PackageTags>Ark;C#;.NET;Framework;AspNetCore;HealthCheck;</PackageTags>
+
+
+		<!--Update these information when releasing a new version of the NugetPackage-->
+		<PackageReleaseNotes></PackageReleaseNotes>
+
+		<Version>$(AspNetCoreGroupVersion)</Version>
+		<VersionSuffix>$(AspNetCoreGroupVersionSuffix)</VersionSuffix>
+
+		<Version Condition=" '$(VersionSuffix)' != '' ">$(Version)-$(VersionSuffix)</Version>
+
+		<AutoGenerateBindingRedirects></AutoGenerateBindingRedirects>
+	</PropertyGroup>
+	<!--===============================================================================-->
+
+	<ItemGroup>
+		<FrameworkReference Include="Microsoft.AspNetCore.App" />
+	</ItemGroup>
+
+	<ItemGroup>
+		<PackageReference Include="AspNetCore.HealthChecks.AzureServiceBus" Version="3.2.2" />
+		<PackageReference Include="AspNetCore.HealthChecks.Network" Version="3.1.3" />
+		<PackageReference Include="AspNetCore.HealthChecks.Oracle" Version="3.1.2" />
+		<PackageReference Include="AspNetCore.HealthChecks.Publisher.ApplicationInsights" Version="3.1.1" />
+		<PackageReference Include="AspNetCore.HealthChecks.RavenDB" Version="3.1.1" />
```

Comment:
Exclude RavenDB dependency from base package, we don't use it often. can be added on Application side

Reply:
Remove the reference from AspNetCore?

Reply:
no, just to RavenDB